cmd/vic-machine/create/create.go

```diff
@@ -864,15 +875,44 @@ func (c *Create) processDNSServers() error {
 	return nil
 }
 
-func (c *Create) processVolumeStores() error {
+func (c *Create) processDefaultVolumeStore() error {
 	defer trace.End(trace.Begin(""))
+	//we make this map here since it is used many times after.
 	c.VolumeLocations = make(map[string]string)
+	if c.DefaultVolumeLocation != "" {
+		c.VolumeLocations[defaultVolumeLabel] = c.DefaultVolumeLocation
+	}
+	return nil
+}
+
+func (c *Create) processVolumeStores() error {
+	defer trace.End(trace.Begin(""))
+
 	for _, arg := range c.volumeStores {
-		splitMeta := strings.SplitN(arg, ":", 2)
-		if len(splitMeta) != 2 {
-			return errors.New("Volume store input must be in format datastore/path:label")
+		var err error
+
+		splitDriverArgs := strings.SplitN(arg, ":", 2)
+		if len(splitDriverArgs) != 2 {
+			return errors.New("volume store input must be in format datastore:label or datastore/path/to/volumestore:label")
+		}
+		label := splitDriverArgs[1]
+		path := splitDriverArgs[0]
+
+		switch label {
+		case defaultVolumeLabel:
+			err = errors.Errorf("multiple paths were tagged for the same label(%s), volumestore labels can only have one distinct path. The preferred method of setting the default store is the --default-volume-store flag.", label)
+		default:
+			err = errors.Errorf("multiple paths were tagged for the same label(%s), volumestore labels can only have one distinct path.", label)
+		}
+
+		otherPath, ok := c.VolumeLocations[label]
```

Comment:
Good catch added a check for this as well. It should not be allowed since it is not helpful or readable it will be filtered out.